src/OrbitGl/TimeGraph.cpp

```diff
@@ -151,24 +151,26 @@ void TimeGraph::ZoomTime(float zoom_value, double mouse_ratio) {
   SetMinMax(min_time_us, max_time_us);
 }
 
-void TimeGraph::VerticalZoom(float zoom_value, float mouse_normalized_y_position) {
+void TimeGraph::VerticalZoom(float zoom_value, float mouse_screen_y_position) {
```

Comment:
Convert `mouse_screen_y_position` to world first (`Viewport::ScreenToWorld`), and then use it in the calculations below. You should never mix world and screen positions in a calculation - this will break if there is no longer a 1:1 mapping between world- and screen coordinates.

Reply:
I really missed that conversion, my bad. Thanks for pointing it out! :)